packages/xo-web/src/common/xo/choose-sr-for-each-vdis-modal/index.js

```diff
@@ -60,6 +60,7 @@ export default class ChooseSrForEachVdisModal extends Component {
     return (
       <div>
         <SelectSr
+          autoSelectSingleOption
```

Comment:
If this select is required, it's better to mark it as such and `autoSelectSingleOption` will be enabled automatically.